mama/c_cpp/src/c/mama/msg.h

```diff
@@ -62,7 +64,8 @@ typedef enum mamaPayloadType_
     MAMA_PAYLOAD_EXEGY      = 'X',
 
 /* Unknown Payload */
-    MAMA_PAYLOAD_UNKNOWN    = 'U'
+    MAMA_PAYLOAD_UNKNOWN    = 'U',
+    MAMA_PAYLOAD_NULL       = '\0'
```

Comment:
its been changed to MAMA_PAYLOAD_ID_NULL now in change (1379ef7345fe2819f14561d9c45a2977b3e42603)